inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -225,12 +221,12 @@ library(rio)
 library(linelist)
 library(janitor)
 library(kableExtra)
-library(incidence2)
 library(grateful)
 library(epiparameter)
+library(incidence2)
 ```
 
-```{r}
+```{r, include=FALSE}
```

Comment:
The reason why I suggested this was to obtain reports that are most meaningful to the reader, while I agree that having chunks in the output that correspond to the outputs of the report, e.g., graphs, tables, etc, I think that some of the other chunks that relate to the style of the template, e.g., `knitr::include_graphics("transmissibility_pipeline.svg")`, will not add any value to the reader, especially if they're not familiar with R or haven't been involved in developing the report, and will maybe even confuse them as to why they're there/what they're doing. I can imagine a field epi or a decision maker being interested in which code/analyses were conducted to obtain a specific figure or estimate, but not really interested in seeing what was the code used to set the theme for the report? I feel like this belongs "behind the scenes" for the reader. 
This doesn't mean they're less important, just that they're not meaningful for a potential reader (vs user) of the report, the user will be able to see these chunks regardless when they're working with the templates- what do you think?

Reply:
Could we skip this change in this PR and open a discussion for wider input for the rest of the team?

In general, `include = FALSE` makes me very uncomfortable because it means report readers won't get the same result if they try to copy/paste the code (even if only changes in colour), which I find very unintuitive for a user point of view.

Reply:
okay I see your point- I will remove this change from the PR for now and open an issue about this

Reply:
Solved in https://github.com/epiverse-trace/episoap/pull/144/commits/367ccc31158d25c380cf63c2af78c5db69ea618f